packages/quantic/cypress/integration/facets/facet/facet-selectors.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  BaseFacetSelector,
+  FacetWithCheckboxSelector,
+  FacetWithSearchSelector,
+  FacetWithShowMoreLessSelector,
+} from '../facet-common-selectors';
+
+export const facetComponent = 'c-quantic-facet';
+
+export type AllFacetSelectors = BaseFacetSelector &
+  FacetWithCheckboxSelector &
+  FacetWithSearchSelector &
+  FacetWithShowMoreLessSelector;
+
+export const FacetSelectors: AllFacetSelectors = {
+  get: () => cy.get(facetComponent),
+
+  label: () => FacetSelectors.get().find('header h2 > span'),
+  values: () => FacetSelectors.get().find('c-quantic-facet-value'),
+  clearButton: () => FacetSelectors.get().find('button[value="Clear"]'),
+  valueLabel: () => FacetSelectors.get().find('.facet__value-text span'),
+  facetValueLabelAtIndex: (index: number) =>
+    FacetSelectors.valueLabel().eq(index),
+  collapseButton: () => FacetSelectors.get().find('.facet__collapse'),
+  expandButton: () => FacetSelectors.get().find('.facet__collapse'),
```

Comment:
is it normal that we have 2 selectors have same value?

Reply:
They are the same button. I think they are duplicated because it makes the test more readable.

Reply:
@nathanlb is right, there is a single button, and tests are more readable this way. We could still toggle between `facet__collapse` and `facet__expand` classes on the same button as well. The selectors would then be different.

Reply:
@lvu285 I'll be fixed as part of [SFINT-4116](https://coveord.atlassian.net/browse/SFINT-4116).